src/OData.QueryBuilder/Conventions/Operators/IODataOperator.cs

```diff
@@ -9,6 +9,8 @@ public interface IODataOperator
 
         bool All<T>(IEnumerable<T> columnName, Func<T, bool> func);
 
+        bool Any<T>(IEnumerable<T> columnName);
+
         bool Any<T>(IEnumerable<T> columnName, Func<T, bool> func);
```

Comment:
I wanted to do `bool Any<T>(IEnumerable<T> columnName, Func<T, bool> func = null);`. But during usage you would get the error `CS0854: An expression tree may not contain a call or invocation that uses optional arguments` during usage.